src/main/flight/mixer.c

```diff
@@ -424,44 +425,79 @@ static void updateDynLpfCutoffs(timeUs_t currentTimeUs, float throttle)
 #endif
 
 static void applyMixerAdjustmentLinear(float *motorMix, const bool airmodeEnabled) {
-    const float motorMixNormalizationFactor = motorMixRange > 1.0f ? motorMixRange : 1.0f;
-    const float motorMixDelta = 0.5f * motorMixRange;
+    float motorMixNormalizationFactor;
+    float airmodeTransitionPercent;
```

Comment:
maybe initialise `airmodeTransitionPercent` to `1.0f` here, and not calculate it to 1.0f in line 433?